c_binding/include/libstoragemgmt/libstoragemgmt_hash.h

```diff
@@ -26,25 +26,25 @@
 extern "C" {
 #endif
 
-/**
+/*
```

Comment:
Looks like this entire header needs to be updated to reflect the new documentation style.